azurerm/resource_arm_mysql_server.go

```diff
@@ -42,12 +41,24 @@ func resourceArmMySqlServer() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
```

Comment:
I still want to remove the duplications here because all the information are expressed in other fields already.

Reply:
Maybe in a future release, right now this is a direct import of the schema.